theorems/I000177.md

```diff
@@ -0,0 +1,14 @@
+---
+if:
+  - P123456
+  - P000016
+then:
```

Comment:
Necessity is the mother of invention. I can never be sure if I've spelled "antecedent" or "consequent" correctly. (Had to use spell check on 1/2 just now.)